dataprofiler/profilers/profile_builder.py

```diff
@@ -866,6 +866,31 @@ def report(self, report_options: dict = None) -> dict:
         """
         raise NotImplementedError()
 
+    @classmethod
+    def load_from_dict(cls, data) -> BaseProfiler:
+        """
+        Parse attribute from json dictionary into self.
+
+        :param data: dictionary with attributes and values.
+        :type data: dict[string, Any]
+
+        :return: Profiler with attributes populated.
+        :rtype: BaseProfiler
+        """
+        profiler = cls(None)
+
+        for attr, value in data.items():
+            if "times" == attr:
+                setattr(profiler, "times", defaultdict(float, value))
```

Comment:
If we set the value in here, it will get overwritten below I believe.
`value = defaultdict(float, value)`

Reply:
is this a problem elsewhere in code

Reply:
👀

Reply:
I am running all the tests with this fix to ensure there is no issue

Reply:
Okay tests seem to be fine with this so it looks like a solidified problem to specific StructuredProfiler